img/seniors/mohit/senior.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Mohit",
+    "profileimg": "./img/seniors/mohit/img/prf.png",
+    "p1_desc": "It was Mohit's first day at BITS Pilani along with a thousand more. Little did they know that this place would create the greatest memories of their lifetime!",
+    "p2_desc": "And since then, their journey at BITS has been somewhat adventurous. But life does not give you the ability to just stick to one thing. As time passes, we must move on...",
+    "imgOnRight": "Path to image",
+    "swiperImages": [
+        "./img/seniors/mohit/img/1.jpeg",
+        "./img/seniors/mohit/img/2.jpeg",
+        "./img/seniors/mohit/img/3.jpeg",
+        "./img/seniors/mohit/img/4.jpeg",
+        "./img/seniors/mohit/img/5.jpeg"
+    ],
+    "txt4": "Like most of us ...",
+    "section6bg": "img",
+    "shortText": "Mohit's a jolly good fellow",
+    "shortText2": "And so say all of us. With all the melodrama of a bollywood potboiler, no less. So turn up the volume, grab your popcorn and let’s go.",
```

Comment:
Replace this with something from the form data and make sure it doesn't overflow from the respective div.